### PR TITLE
Supports screenOrientation in TWAS

### PR DESCRIPTION
**Pull request by sclimwi**
1) Add a field in the LauncherActivity called android:screenOrientation to lock the splashScreen
2) Add a meta data in the AndroidManifest.xml in order to lock all the activities' into a lockType
3) Add a @Nullable string field in LanucherActivityData called screenOrientation
4) Add the .setScreenOrientation() method for the twaActivityIntentBuilder, which will be implemented in the androidX part
5) Add a new demo called twa-nofification-delegation-orientation so we can test the orientation locking without changing other demos

I think I did not do the properly, but this version looks cleaner. 
**I need to figure out how to push to the same PR, instead of creating new PR every time (:.** 

 @PEConn @andreban @glennhartmann @EiraGe

### CODE REVIEW

**Comment by sclimwi** [comment]
Looks like I should follow the following procedure from now on, in order to add changes into the same PR? 
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by PEConn** [comment]
Yeah, so I think what you did was create a new branch and made the changes there - in general it's better to make the changes on the same branch you used previously and upload a new patch, that way we can see what you've changed since we last commented on stuff.

File: demos/twa-notification-delegation-orientation/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Trusted Web Activity / Notification Delegation Demo
```
Comment (PEConn) [inline demos/twa-notification-delegation-orientation/README.md:1]: How come the demo you've made is called `twa-notification-delegation-orientation`? It looks like you've mostly copied the notification delegation demo (which is fine), but not got rid of the notification delegation parts.

What I'd recommend is that you copy from the `twa-basic` instead (it's the simplest one) and then name it `twa-landscape` or `twa-orientation` or something like that.

File: demos/twa-notification-delegation-orientation/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.LauncherActivity">
+        <meta-data
+            android:name="asset_statements"
+            android:resource="@string/asset_statements" />
+
+        <activity android:name="com.google.androidbrowserhelper.trusted.LauncherActivity"
+            android:label="@string/app_name" android:screenOrientation="landscape">
```
Comment (PEConn) [inline demos/twa-notification-delegation-orientation/src/main/AndroidManifest.xml:29]: nit: put `android:label` and `android:screenOrientation` on different lines.

File: androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java [hunk 3]
```diff
@@ -35,6 +36,8 @@
  */
 public class LauncherActivityMetadata {
 
+    private static final String TAG = "LauncherActivityMetadata";
```
Comment (PEConn) [inline androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java:39]: We don't use this anywhere (nor the `import`).

Comment (sclimwi) [reply androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java:39]: I will remove this later. I was trying to test the result there.

File: demos/twa-notification-delegation-orientation/src/main/AndroidManifest.xml (hunk 2 above)
Comment (sclimwi) [reply demos/twa-notification-delegation-orientation/src/main/AndroidManifest.xml:29]: Will make the change later.

File: demos/twa-notification-delegation-orientation/README.md (hunk 1 above)
Comment (sclimwi) [reply demos/twa-notification-delegation-orientation/README.md:1]: Yeah. Andre said we do not want to just modify the twa-basic demo as some users may not want to touch with orientation. So I copied the content from the notification to support orientation. 

The reason I chose the notification one is that the website for the twa-basic (svgomg) is not working now.

**Comment by andreban** [comment]
@sclimwi Can you confirm if this PR is the latest? I'll keep only the PR with the latest open and close the other one.

**Comment by sclimwi** [comment]
> @sclimwi Can you confirm if this PR is the latest? I'll keep only the PR with the latest open and close the other one.

Yes. This is the latest. Thanks.

**Comment by PEConn** [comment]
Hey @sclimwi , I'm not quite sure what's going on, but you have almost 400 changed files in this PR.

**Comment by sclimwi** [comment]
> Hey @sclimwi , I'm not quite sure what's going on, but you have almost 400 changed files in this PR.

Hey, @PEConn. I saw that. I was trying to upload the new changes and renamed the demo to be twa-orientation from a previous long name. And I was also trying to squash commits together. All of sudden, everything is there (old and new demo files). I am thinking about either to clean them or just abandon this PR and create a new branch. Then redo the whole thing.

**Comment by PEConn** [comment]
Yeah, maybe a new PR is the best way to go.

**Comment by EiraGe** [comment]
@sclimwi I think the extra changed files is because you didn't add the `build` folder in `.gitignore` file of your demo. (you can find examples in other demos too)

**Comment by sclimwi** [comment]
> @sclimwi I think the extra changed files is because you didn't add the `build` folder in `.gitignore` file of your demo. (you can find examples in other demos too)

Yeah. Sounds right. I did not remember I had that. I will give it a try after I fix the codes in chromium. Trying to push that part out soon.

**Comment by andreban** [comment]
Closing this in favour of #127

**Pull request closed without merging**